Assets/Scripts/Editor/BuildHelpers.cs

```diff
@@ -0,0 +1,235 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+
+/// <summary>
+/// Utility menus to easily create our builds for our playtests. If you're just exploring this project, you shouldn't need those. They are mostly to make
+/// multiplatform build creation easier and is meant for internal usage.
+/// </summary>
+internal static class BuildHelpers
+{
+    const string k_MenuRoot = "Boss Room/Playtest Builds/";
+    const string k_Build = k_MenuRoot + "Build";
+    const string k_DeleteBuilds = k_MenuRoot + "Delete All Builds (keeps cache)";
```

Comment:
Is there any benefits in having a "Delete All Builds and cache"? Or we always want to keep the cache?

Reply:
mmm I'd say we can add this in a followup PR if we need it and if we see there's issues with local build caching? This is for local usage for now with no automation, we can still do things manually if need be. But yes do let me know if you'd like to see improvements for this in the future when using it.